packages/clerk-js/src/ui/customizables/elementDescriptors.ts

```diff
@@ -97,6 +97,7 @@ export const APPEARANCE_KEYS = containsAllElementsConfigKeys([
   'userButtonTrigger',
   'userButtonAvatarBox',
   'userButtonAvatarImage',
+  'userButtonTopLevelIdentifier',
```

Comment:
We already have a `userButtonOuterIdentifier` - maybe use that? Is it used anywhere?

Reply:
Uhm, no it isn't. I guess it just wasn't added to the element back then. Which name do we like better?

Reply:
Probably the outer one, as we do have outer/inner in other places as well

Reply:
I'll just leave the old one since it theoretically could be considered a breaking change to remove it :)